en/xml/fo/topic.graphic-pageheader.xml

```diff
@@ -0,0 +1,232 @@
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.block1">
+                        <para>A <tag>fo:block</tag>-element creates a rectangle
+                            area on a page which can contain lists, tables or
+                            graphics. In this demo, the document title is
+                            visible on the left side of single-sided documents.</para>
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.center">
+                        <para>In this demo, the center position has no content,
+                            but you can add a page number for example.</para>
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.right">
+                        <para>This is the place for our logo as described at the
+                            top of our solution.</para>
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.height">
+                        <para>To avoid conflicts with the body text area, we
+                            limit its height to 10mm. When you specify only
+                            one parameter like <tag>height</tag> or
+                                <tag>width</tag>, your logo will be scaled with
+                            the <link xlink:href="#li.fo.external-graphics"
+                                >correct aspect ratio</link>.</para>
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.template">
+                        <para>The template which is called to import and place
+                            the logo.</para>
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.attribute">
+                        <para>The file name is passed on the <tag
+                                class="attribute">src</tag> attribute.</para>
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.file">
+                        <para>Our file defined at the top of our customization
+                            layer.</para>
+                    </callout>
+                </calloutlist>
+            </step>
+            <step>
+                <para>Use your customization layer to transform your DocBook
+                    document into FO.</para>
+            </step>
+        </procedure>
+    </section>
+    <section role="discussion">
+        <title>Discussion</title>
+        <para>The stylesheet <filename>graphic-pageheader.xsl</filename> creates
+            a logo in the right edge of your page and uses the parameter
+                <parameter>logo.src.path</parameter> to find the logo. By
+            default, a logo is stored in <filename>logo/</filename>. If your
+            logos are in a different location, change it with your XSLT
```

Comment:
Can there be plural (logo**s**)?  Isn't only one logo used/placed in the header/footer?